src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/CraftPriceTooltip.java

```diff
@@ -0,0 +1,108 @@
+package de.hysky.skyblocker.skyblock.item.tooltip.adders;
+
+import de.hysky.skyblocker.config.SkyblockerConfigManager;
+import de.hysky.skyblocker.config.configs.GeneralConfig;
+import de.hysky.skyblocker.skyblock.item.tooltip.ItemTooltip;
+import de.hysky.skyblocker.skyblock.item.tooltip.TooltipAdder;
+import de.hysky.skyblocker.skyblock.item.tooltip.TooltipInfoType;
+import de.hysky.skyblocker.utils.NEURepoManager;
+import io.github.moulberry.repo.data.NEUIngredient;
+import io.github.moulberry.repo.data.NEUItem;
+import io.github.moulberry.repo.data.NEURecipe;
+import net.minecraft.item.ItemStack;
+import net.minecraft.screen.slot.Slot;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class CraftPriceTooltip extends TooltipAdder {
+    protected static final Logger LOGGER = LoggerFactory.getLogger(CraftPriceTooltip.class.getName());
+    private final static Map<String, Double> cachedCraftCosts = new ConcurrentHashMap<>();
+    private static final int MAX_RECURSION_DEPTH = 15;
+
+    public CraftPriceTooltip(int priority) {
+        super(priority);
+    }
+
+    @Override
+    public void addToTooltip(@Nullable Slot focusedSloFt, ItemStack stack, List<Text> lines) {
+        if (SkyblockerConfigManager.get().general.itemTooltip.enableCraftingCost == GeneralConfig.Craft.OFF) return;
+
+        String internalID = stack.getSkyblockId();
+        if (stack.getNeuName() == null || internalID == null) return;
+
+        if (TooltipInfoType.LOWEST_BINS.getData() == null || TooltipInfoType.BAZAAR.getData() == null) {
+            ItemTooltip.nullWarning();
+            return;
+        }
+
+        NEUItem neuItem = NEURepoManager.NEU_REPO.getItems().getItemBySkyblockId(internalID);
+        if (neuItem == null) return;
+
+        List<NEURecipe> neuRecipes = neuItem.getRecipes();
+        if (neuRecipes.isEmpty() || neuRecipes.getFirst().getClass().equals(io.github.moulberry.repo.data.NEUKatUpgradeRecipe.class)) return;
+
+        try {
+            double totalCraftCost = getItemCost(neuRecipes.getFirst(), 0);
+
+            if (totalCraftCost == 0) return;
+
+            neuRecipes.getFirst().getAllOutputs().stream().findFirst().ifPresent(neuIngredient ->
+                    lines.add(Text.literal(String.format("%-20s", "Crafting Price:")).formatted(Formatting.GOLD)
+                            .append(ItemTooltip.getCoinsMessage(totalCraftCost / neuIngredient.getAmount(), stack.getCount()))));
```

Comment:
Could copy the sack price multiplication logic from `BazaarTooltip` and use that instead of `stack.getCount()` for the crafting price to be multiplied according to the amount you have in the sack.

Reply:
I thought that was being done by `neuIngredient.getAmount()`, nvm then.

Reply:
Wait, I think there's a misunderstanding.
You're still multiplying the price message by the amount of items in the stack without this change. So if you have 10 items in your inventory that are craftable, you'll see the price to craft all 10 rather than 1. The fix I mentioned makes it work on the sack as well, so if you have like 100 enchanted cookies you see the craft cost of 100 rather than just 1.
This isn't related to the division from the output count from the crafting result